### ISSUE TITLE
Update pom.xml on Maven Central

### ISSUE THREAD

**Issue by simonedavico** [jcazevedo/moultingyaml#8]
First things first, thanks for the amazing library, working with Scala and YAML is less painful now!

On to the unpleasant matters: could it be possible to update the `pom.xml` file for the library on Maven Central?

As of today, this dependency

```
<dependency>
            <groupId>com.github.nscala-time</groupId>
            <artifactId>nscala-time_2.11</artifactId>
            <version>2.0.0</version>
</dependency>
```

requires Scala `2.11.6`; this results in multiple Scala libraries downloaded in the project in which I am using MoultingYAML. Could it be updated to `2.2.0`, as in your `sbt` file?

**Comment by simonedavico** [comment]
I just realised that forcing the latest version by specifying it as a dependency in my POM solves the problem. Still, it would be nice to have it updated.

**Comment by jcazevedo** [comment]
I've just released version 0.2 of MoultingYAML with the updated dependencies to Maven Central. Please update your MoultingYAML version to 0.2.

**Comment by simonedavico** [comment]
Thank you, much appreciated. I'm closing the issue then.
